### ISSUE TITLE
Pseudo-elements failing on Table Row with x-target and [aria-busy]

### ISSUE THREAD

**Issue by Jagadish056** [imacrayon/alpine-ajax#94]
When using both a [Loading Indicator](https://alpine-ajax.js.org/examples/loading/) and [Edit Row](https://alpine-ajax.js.org/examples/edit-row/) on the same page, I've noticed that pseudo-elements are not functioning correctly on the `tr` (table row) element. Specifically, this issue occurs with the `x-target` and `[aria-busy]` attributes.

Example screenshots:

![Screenshot 2024-09-10 145521](https://github.com/user-attachments/assets/714493a6-211b-424a-a132-5cdbe0ba46c4)
![Screenshot 2024-09-10 145613](https://github.com/user-attachments/assets/3319e92f-3b97-4cc6-b256-2e90a13c8bf1)

Highlighting the issue:
![Screenshot 2024-09-10 145614](https://github.com/user-attachments/assets/15e5cae9-36e6-4ddb-b805-eab7e481ea35)

**Potential Solution:** A workaround seems to be using either `:after` pseudo-elements or targeting the `tr[aria-busy] td:first-child::before` selector. 

[Is it possible to use pseudo-elements (:after, :before) inside a table row?](https://stackoverflow.com/questions/19588674/is-it-possible-to-use-pseudo-elements-after-before-inside-a-table-row)

Does this approach align with the intended behavior, or is there a more efficient way to handle this scenario?

**Comment by imacrayon** [comment]
I think targeting the first `<td>` makes sense, I've seen this approach in Tailwind UI. Unfortunate this is a common limitation around HTML nesting.

**Comment by imacrayon** [comment]
I'm gonna close this out, but if you have any other questions/comment feel free to reopen. We're also working on a new API for loading states that might provide some more flexibility in situations like this soon.
